pyrolite/geochem/transform.py

```diff
@@ -349,8 +349,10 @@ def aggregate_element(
 
     if logdata:
         logger.debug("Log-transforming {} Data.".format(cation))
-        _df.loc[:, targetnames] = _df.loc[:, targetnames].applymap(np.log)
-
+        try:
```

Comment:
In this case where the function (`np.log`) being used is clear, I think `numpy` and `pandas` play nicely, ` _df.loc[:, targetnames] = np.log(_df.loc[:, targetnames])` should suffice - and be faster.